reference/conanfile/methods.rst

```diff
@@ -71,26 +75,36 @@ You can also use **check_md5**, **check_sha1** and **check_sha256** from the **t
                 else:
                     # download sources from Nix systems in a tgz
 
-    If you need to patch the source code or build scripts differently for different variants of your packages, you can do it in the ``build()`` method, which uses a different folder and source code copy for each variant.
-
+    If you need to patch the source code or build scripts differently for different variants of your packages, you can do it in the
+    ``build()`` method, which uses a different folder and source code copy for each variant.
 
 build()
 -------
 
+This method is used to build the source code of the recipe using the desired commands. You can use your command line tools to invoke your
+build system or any of the build helpers provided with Conan.
+
+.. code-block:: python
+
+    def build(self):
+        cmake = CMake(self)
+        self.run("cmake . %s" % (cmake.command_line))
```

Comment:
Better use modern ``cmake.configure()`` syntax.

Reply:
``self.run()`` was left indentenly to show the ussage of both build helpers and normal command line instructions.

Reply:
Good, leave it.